acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -119,89 +118,55 @@ def pxp_agent_config_file(host)
 
 # @param host a beaker host (to determine the correct path for the OS)
 # @param client_number which client 01-05 you want the key file for
+# @param [boolean] use_alt_path use alt file path?
```

Comment:
Yard will introspect the default from the method definition. Each method only defines a single path, not a set of certs. Maybe the following is more accurate?

    # @param [boolean] use_alt_path Use alternative test ssl file if true, otherwise use standard test ssl file